File: 191. Number of 1 Bits.cpp
class Solution {
public:
    int hammingWeight(uint32_t n) {
        string s = bitset <32> (n).to_string();
        int res = 0;
        for (int i = 0; i < s.size(); i++)
            res += s[i] - '0';
        return res;
    }
};

File: 191. Number of 1 Bits.js
/**
 * @param {number} n - a positive integer
 * @return {number}
 */
var hammingWeight = function(n) {
    let s = (n >>> 0).toString(2);
    let res = 0;
    for (let i = 0; i < s.length; i++)
    {
        if (s[i] == '1')
            res += 1;
    }
    return res;
};
